libs/live-fns/keybase.js

```diff
@@ -0,0 +1,29 @@
+const got = require('../got.js')
+
+const convertFingerprintTo64bit = fingerprint => fingerprint.slice(fingerprint.length - 16).toUpperCase().match(/.{1,4}/g).join(' ')
```

Comment:
This can be `.slice(-16)`

Reply:
Done https://github.com/amio/badgen-service/pull/141/commits/3d8ca72df11b1f5a92289d9dea9d8e35922a0b32